### ISSUE TITLE
Dependency on Logging.jl

### ISSUE THREAD

**Issue by MaximeBouton** [JuliaPOMDP/RLInterface.jl#7]
Is this dependency necessary? By looking at the source code it seems like it was useful for debugging purposes and could potentially be removed. 

The main motivation is that it causes warning when loading the package: 
```julia
julia> using Logging
WARNING: Method definition info(Any...) in module Base at util.jl:532 overwritten in module Logging at C:\Users\Maxime\.julia\v0.6\Logging\src\Logging.jl:115.
WARNING: Method definition warn(Any...) in module Base at util.jl:585 overwritten in module Logging at C:\Users\Maxime\.julia\v0.6\Logging\src\Logging.jl:115.
```

The ideal solution would probably be to submit a PR to Logging.jl

**Comment by rejuvyesh** [comment]
v0.7 will have great Logging built in. We can stop using `Logging.jl`

**Comment by MaximeBouton** [comment]
You are right, we can replaced it by `@info` `@warn` and so on once we switch to 0.7 then

**Comment by MaximeBouton** [comment]
fixed #11
